src/sdcv.cpp

```diff
@@ -186,10 +186,11 @@ try {
         }
 
         // add bookname to list
-        gchar **p = get_impl(use_dict_list);
-        while (*p) {
-            order_list.push_back(bookname_to_ifo.at(*p));
-            ++p;
+        for (gchar **p = get_impl(use_dict_list); *p != nullptr; ++p) {
+            auto it = bookname_to_ifo.find(*p);
+            if (it != bookname_to_ifo.end()) {
+                order_list.push_back(it->second);
+            }
```

Comment:
What about report to stderr and return EXIT_FAILURE if there are no such dictionary instead of silently ignore error in user supplied list?

Reply:
That sounds sensible, but raises another question: should we abort early on *any* failure, or continue with a potentially limited dictionary list (if any)?

(Warning no matter what makes sense, though).